### ISSUE TITLE
`test_sbt`: mute regular console output from DefaultTestRunner

### ISSUE THREAD

**Issue by dkarlinsky** [zio/zio#1403]
`RunnableSpec`s come wired with their own `TestRunner` which has a `TestReporter`. 
This works great for running them via their object's `main` method, but under `sbt` it generates duplicated output like the following:
```
+ ZQueueSpec
  + simple offer and take
  + simple offer and take 2
[info] + ZQueueSpec
[info]   + simple offer and take
[info]   + simple offer and take 2
[info] ZIO Test
[info] Done
[info] Passed: Total 2, Failed 0, Errors 0, Passed 2
```
SBT test runner should probably run specs with a custom "muted" `TestReporter`, to prevent this.
cc: @ghostdogpr

**Comment by dkarlinsky** [comment]
PR to follow soon

**Comment by ghostdogpr** [comment]
Fixed by #1405
